src/utils/parse.js

```diff
@@ -89,8 +122,12 @@ export const generateSampleTable = response => {
 
   response.forEach(run => {
     const iterations = {};
-    const id = run.aggregations.id.buckets[0].key;
     const primaryMetrics = new Set();
+
+    if (run.hits.hits.length === 0) {
+      return;
```

Comment:
Is this a condition that implies an error occurred, or is this indicated on the page somehow?